public/router.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var s3uploader = angular.module('s3Uploader', ['ui.router'])
+  .config(['$stateProvider', '$urlRouterProvider', function ($stateProvider, $urlRouterProvider) {
+    $urlRouterProvider.otherwise('/');
+    $stateProvider
```

Comment:
What is the busineess with code in 'public/file-upload/file-upload.js'  both seem to look same?